charts/terraform-operator-remote-controller/Chart.yaml

```
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: 1.3.3
```

```suggestion
appVersion: 1.4.0
```

Also values version of the image should be `1.4.0`